workers/data_refinery_workers/processors/array_express.py

```diff
@@ -160,8 +160,15 @@ def _create_result_objects(job_context: Dict) -> Dict:
     result.time_start = job_context['time_start']
     result.time_end = job_context['time_end']
     result.pipeline = "Affymetrix SCAN"  # TODO: should be removed
-    # result.processor = Processor.objects.get(name=utils.ProcessorEnum.AFFYMETRIX_SCAN.value,
-    #                                          version=__version__)
+    try:
+        result.processor = utils.find_processor("AFFYMETRIX_SCAN")
+    except Exception as e:
+        err_str = "Failed to set processor: %s" % e
+        logger.error(err_str)
```

Comment:
Same comment about additional context for this logger call and the other ones just like it.